.github/workflows/backport.yml

```diff
@@ -0,0 +1,25 @@
+name: backport
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  backport:
+    name: Backport PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Backport Action
+        uses: sorenlouv/backport-github-action@v9.3.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we have to worry about sorenlouv (hypothetically) stealing or secret token?

Reply:
lol that's a valid concern, maybe we should fork it?

Reply:
Looks fairly trusted.  Meh.  I think we should have 
`uses: sorenlouv/backport-github-action@f7073a2287aefc1fa12685eb25a712ab5620445c # v9.3.0`
instead to pin to a commit then.  Less prone to unwitting hijacks.

Reply:
It seems this `github_token` will be whoever merges this pr?

Reply:
Oh, I meant to use the commit hash if we don't fork.  If we fork then tags are under our control  :) NBD
I'm just talking through security concerns since I don't really know how the action works.
I suspect this guy's project is fine if we pin by commit hash instead of something changeable like a tag or latest or whatever.

Reply:
haha, just incase the repo could be gone...